crates/autopilot/src/domain/settlement/solution/trade.rs

```diff
@@ -147,12 +163,10 @@ impl Trade {
                 .ok_or(error::Math::Overflow)?
                 .checked_div(&self.prices.uniform.sell.into())
                 .ok_or(error::Math::DivisionByZero)?,
-        };
-
-        let price = prices
-            .get(&self.sell.token)
-            .ok_or(Error::MissingPrice(self.sell.token))?;
-        Ok(price.in_eth(fee_in_sell_token))
+        }
+        .0
```

Comment:
```suggestion
        
```

Reply:
nice catch :) forgot to update after adding From implementation